<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpectralProcessingWidget</class>
 <widget class="QWidget" name="SpectralProcessingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectral Processing Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
       <property name="title">
        <string>Spectral Algorithms</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="SpectralProcessingAlgorithmTreeView" name="mTreeView">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragOnly</enum>
          </property>
          <attribute name="headerVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="verticalGroupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Model</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="SpectralProcessingModelTableView" name="mTableView">
          <property name="dragEnabled">
           <bool>true</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>429</width>
         <height>487</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddFunction">
   <property name="icon">
    <iconset resource="../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionAddExpression.svg</normaloff>:/images/themes/default/mActionAddExpression.svg</iconset>
   </property>
   <property name="text">
    <string>Add Function</string>
   </property>
   <property name="toolTip">
    <string>Adds a new Spectral Math function</string>
   </property>
  </action>
  <action name="actionRemoveFunction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionRemove.svg</normaloff>:/images/themes/default/mActionRemove.svg</iconset>
   </property>
   <property name="text">
    <string>Remove Function</string>
   </property>
   <property name="toolTip">
    <string>Removes the selected function(s)</string>
   </property>
  </action>
  <action name="actionLoadModel">
   <property name="icon">
    <iconset resource="../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionFileOpen.svg</normaloff>:/images/themes/default/mActionFileOpen.svg</iconset>
   </property>
   <property name="text">
    <string>Load Model</string>
   </property>
  </action>
  <action name="actionSaveModel">
   <property name="icon">
    <iconset resource="../../../QGIS/images/images.qrc">
     <normaloff>:/images/themes/default/mActionFileSave.svg</normaloff>:/images/themes/default/mActionFileSave.svg</iconset>
   </property>
   <property name="text">
    <string>Save Model</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SpectralProcessingAlgorithmTreeView</class>
   <extends>QTreeView</extends>
   <header>qps.speclib.processing</header>
  </customwidget>
  <customwidget>
   <class>SpectralProcessingModelTableView</class>
   <extends>QTableView</extends>
   <header>qps.speclib.processing</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../QGIS/images/images.qrc"/>
 </resources>
 <connections/>
</ui>
